values/values.go

```diff
@@ -148,6 +152,13 @@ func New(v interface{}) Value {
 	}
 }
 
+func NewNull(t semantic.Type) Value {
```

Comment:
this is great.  I'm getting closer to voting that we can only add values.Value  types to table builders, to make sure we don't have any errors of omission re: Nulls.